commit ff1092b42145559be68e84c3c22ae919124f2d4e (refs/changes/06/4906/2)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-10-02T12:16:24-04:00 (1 year ago)
    
    OS-7273 vminfo manpage should be in gitignore
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/.gitignore b/.gitignore
index 24bf3ef3..ccb2a5ec 100644
--- a/.gitignore
+++ b/.gitignore
@@ -68,6 +68,7 @@ man/man/man1m/sysinfo.1m
 man/man/man1m/diskinfo.1m
 man/man/man1m/disklayout.1m
 man/man/man1m/mkzpool.1m
+man/man/man1m/vminfo.1m
 man/man/man5/fwrule.5
 src/cryptpass
 src/diskinfo
